index.js

```diff
@@ -1,55 +1,386 @@
 const state = {
   items: [
     {
-      id: "001-beetroot",
+      id: "001-beetroot.svg",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
+      type: "vegetable",
     },
     {
-      id: "002-carrot",
+      id: "002-carrot.svg",
       name: "carrot",
-      price: 0.35
+      price: 0.45,
+      type: "vegetable",
     },
     {
-      id: "003-apple",
+      id: "003-apple.svg",
       name: "apple",
-      price: 0.35
+      price: 0.65,
+      type: "fruit",
     },
     {
-      id: "004-apricot",
+      id: "004-apricot.svg",
       name: "apricot",
-      price: 0.35
+      price: 0.41,
+      type: "fruit",
     },
     {
-      id: "005-avocado",
+      id: "005-avocado.svg",
       name: "avocado",
-      price: 0.35
+      price: 0.52,
+      type: "fruit",
     },
     {
-      id: "006-bananas",
+      id: "006-bananas.svg",
       name: "bananas",
-      price: 0.35
+      price: 0.55,
+      type: "fruit",
     },
     {
-      id: "007-bell-pepper",
+      id: "007-bell-pepper.svg",
       name: "bell pepper",
-      price: 0.35
+      price: 0.65,
+      type: "vegetable",
     },
     {
-      id: "008-berry",
+      id: "008-berry.svg",
       name: "berry",
-      price: 0.35
+      price: 0.33,
+      type: "fruit",
     },
     {
-      id: "009-blueberry",
+      id: "009-blueberry.svg",
       name: "blueberry",
-      price: 0.35
+      price: 0.72,
+      type: "fruit",
     },
     {
-      id: "010-eggplant",
+      id: "010-eggplant.svg",
       name: "eggplant",
-      price: 0.35
-    }
+      price: 0.42,
+      type: "vegetable",
+    },
   ],
-  cart: []
-};
\ No newline at end of file
+  cart: [],
+};
+
+const storeItems = state.items;
+const cart = state.cart;
```

Comment:
Be aware that these 2 variables contain references to the original object properties